src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -515,17 +518,7 @@ void AtlasEngine::_handleSettingsUpdate()
         _recreateCellCountDependentResources();
     }
 
-    // !!! NOTE !!!
-    // This will indirectly mark the entire viewport as dirty, but AtlasEngine::_recreateBackend()
-    // may do the same, so make sure that that function stays in sync with what this code does.
     _api.invalidatedRows = invalidatedRowsAll;
-
-    u16 y = 0;
-    for (const auto r : _p.rows)
-    {
-        r->clear(y, _p.s->font->cellSize.y);
-        ++y;
-    }

```

Comment:
This was previously here, because `clear()` resets the dirty area of each row. When the settings change, it should reset the dirty area, because - for example - if the font changes, the `deltaPx` by which each row gets "scrolled" will be wrong, relative to the previous height of each row.

But now I've realized that this doesn't actually matter. No matter what incorrect calculations we make, _everything's invalid anyways_. It literally can't get any more invalid, no matter whether I clear here or not!